src/tab/Tab.tsx

```diff
@@ -0,0 +1,48 @@
+import "./_tab.scss";
+
+import React, {useState} from "react";
+import classNames from "classnames";
+
+import Button from "../button/Button";
+
+export type TabItem = {
+  id: string;
+  children: React.ReactNode;
+  icon?: React.ReactNode;
+};
+
+interface TabProps {
+  items: TabItem[];
+  children: React.ReactNode[];
+  customClassName?: string;
+}
+
+function Tab({items, children, customClassName}: TabProps) {
+  const [activeTabIndex, setActiveTabIndex] = useState(0);
+  const tabClassName = classNames("tab", customClassName);
+
+  return (
+    <div className={tabClassName}>
+      <header className={"tab-header"}>
+        {items.map((item, index) => (
+          <Button
+            key={item.id}
```

Comment:
We should probably be adding `testids` otherwise we will get complaints from the QA team :)